### PR TITLE
Changed needed to run "restructure" workflow on Nurion

### PR DESCRIPTION
**Pull request by sungeunbae**
#### Description
restructure PR is pending "merge" (awaiting @jpa198 's approval)
In the mean time, branched out this one to add changes needed for Nurion.

#### Checklist
Will update the following later.

- Have you updated the README (yes/no)? 
- Have you updated the CHANGELOG (yes/no)?

### CODE REVIEW

File: workflow/automation/estimation/estimate_wct.py [hunk 1]
```diff
@@ -469,6 +469,8 @@ def est_IM_chours_single(
             (coefficients["a"] * np.log(nt * fd_count * comp_count)) + coefficients["b"]
         )
 
+    if config.host == 'nurion':
```
Comment (jasonmotha) [inline workflow/automation/estimation/estimate_wct.py:472]: Would be good to check `config` wasn't none / had host has an attribute too.

**Review by jasonmotha — COMMENTED** [review]
Generally happy with this. Is it meant to merge into restructure or master?

**Comment by sungeunbae** [comment]
> Generally happy with this. Is it meant to merge into restructure or master?

It's branched off from restructure. But now that restructure has been merged into master, I think either way is fine.

**Review by jpa198 — APPROVED** [review]

**Comment by sungeunbae** [comment]
https://github.com/ucgmsim/slurm_gm_workflow/pull/422#discussion_r780843016 
Cool. I will add that info and keep it commented out.

**Comment by sungeunbae** [comment]
@jasonmotha  Will you have another look?

**Comment by sungeunbae** [comment]
@jasonmotha  See https://github.com/ucgmsim/slurm_gm_workflow/pull/422/commits/b074ebc20323cfb33e6b3e65c8251882a8b08fef

**Review by jasonmotha — APPROVED** [review]

**Comment by jasonmotha** [comment]
@sungeunbae Just as a heads up, you'll need to merge restructure back to master if you want these changes in master too.

**Comment by sungeunbae** [comment]
> @sungeunbae Just as a heads up, you'll need to merge restructure back to master if you want these changes in master too.

@jasonmotha  Indeed!

**Pull request merged**